### DISCUSSION TITLE
Add a way to manually disable the cinnamon compositor

### DISCUSSION THREAD

**Discussion by awsms** [linuxmint/discussions#169] [Ideas]
Hi! I'd like to know if there are any plans to re-add a way to manually disable the muffin compositor (mainly for gaming with wine).

https://linux-gaming.kwindu.eu/index.php?title=Compositor_(X11)
So far, only cinnamon doesn't let you disable the compositor on-demand :(

**Comment by rcalixte** [comment]
Have you tried the setting under System Settings -> General -> Compositor Options -> Disable compositing for full-screen windows? That page suggests it might need updating as well.

**Reply by awsms** [reply]
Hi Rick, indeed I already have this option enabled.
I've opened another discussion regarding this option, since I'm not sure if it's used as well for fullscreen toggled windows (#170).
I play games as borderless windows I then maximize to fullscreen with the cinnamon shortcut, since Alt+Tab often freezes the Vulkan renderer if the window is in exclusive fullscreen mode.
